src/middleware/permissionsMiddleware.test.js

```diff
@@ -0,0 +1,21 @@
+import { describe, it } from 'mocha'
+import { expect } from 'chai'
+
+
+describe('query', () => {
+  describe('statistics', () => {
+    describe('authenticated user', () => {
+      describe('read', () => {
+        it('is forbidden', () => {
```

Comment:
@appinteractive there are meaningful tests missing here, don't merge

Reply:
@roschaefer question is if we should write the tests here or in a separate ticket?